openqa-label-known-issues

```diff
@@ -17,6 +17,8 @@ issue_marker="${issue_marker:-"auto_review%3A"}"
 issue_query="${issue_query:-"https://progress.opensuse.org/projects/openqav3/issues.json?limit=200&subproject_id=*&subject=~${issue_marker}"}"
 reason_min_length="${reason_min_length:-"8"}"
 grep_timeout="${grep_timeout:-5}"
+email_unreviewed=${email_unreviewed:-true}
+from_email=${from_email:-okurz@suse.de}
```

Comment:
Why me? :)

How about

```suggestion
from_email=${from_email:-openqa-label-known-issues@open.qa}
```

which is of course not a valid email address and we should select something infrastructure specfiic wherever we use it

Reply:
done